ts/DataGrid/DataGridOptions.d.ts

```diff
@@ -21,21 +27,82 @@ import type DataTable from '../Data/DataTable';
  *
  * */
 
+/**
+ * Options to control the content and the user expierence of a grid structure.
+ */
 export interface DataGridOptions {
+
+    /**
+     * Height of each grid cell. This is used to calculate the amount of visible
+     * cells in a container and the size of the scrollbar.
+     *
+     * @default 25
+     */
     cellHeight: number;
+
+    /**
+     * Contains options for column headers.
+     */
     columnHeaders: ColumnHeaderOptions;
+
+    /**
+     * Contains column-specific options. The key is the column name and the
+     * value is the object with the column-specific options.
+     */
     columns: Record<string, ColumnOptions>;
+
+    /**
+     * Table data to display in the grid structure.
+     */
     dataTable?: DataTable;
+
+    /**
+     * Switch to make the whole grid structure with all cells editable ('true')
+     * or read-only ('false').
+     *
+     * @default true
+     */
     editable: boolean;
+
+    /**
+     * Switch to make the column sizes editable ('true') or fixed ('false').
+     *
+     * @default true
+     */
     resizableColumns: boolean;
 }
 
+/**
+ * Contains options for column headers.
```

Comment:
Do we have to repeat the description here? What if they mismatch?

Reply:
What do you have in mind?

Reply:
In lines 43-46, you have a description for the `  columnHeaders?: ColumnHeaderOptions;` and also the same description is in lines 75-77. I was wondering which is truly picked up by the typedoc for documentation.

But as far as I can see we might have a different description in both places.
You can ignore this request ;)